### ISSUE TITLE
cl events doesn't work due to argument parsing

### ISSUE THREAD

**Issue by percyliang** [codalab/codalab-worksheets#263]
`cl events` doesn't work properly (returns empty) because args.command gets set to 'events' for some reason instead of None.  Seems to be a problem with the command-line arguments parsing.

**Comment by kashizui** [comment]
I can't seem to reproduce the error, I see this when running `cl events`:

```
2015-11-30 11:57:00 0.012   sckoo2(1)   get_target_info [["0xf86376d85867477280af87963c34e5f8", ""], 2]
2015-11-30 11:57:00 0.011   sckoo2(1)   head_target [["0xf86376d85867477280af87963c34e5f8", ""], 100]
2015-12-01 15:16:54 0.000   None(0) start_bundle    ["0x3c3c27d123ed4559a6ae663f733ae40a"]
2015-12-01 15:16:55 0.000   None(0) finalize_bundle ["0x3c3c27d123ed4559a6ae663f733ae40a", "failed", {"job_handle": "35311", "last_updated": 1449011814, "data_size": 248, "exitcode": 127}]
2015-12-01 15:16:55 0.000   None(0) start_bundle    ["0xaf33b205771d402e82dad59964b0c6c6"]
2015-12-01 15:16:55 0.000   None(0) finalize_bundle ["0xaf33b205771d402e82dad59964b0c6c6", "ready", {"job_handle": "35322", "last_updated": 1449011814, "data_size": 140, "exitcode": 0}]
2015-12-01 15:16:57 0.009   sckoo2(1)   search_worksheets   [["tag=paper,software,data"]]
2015-12-01 15:16:58 0.055   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", false, true]
2015-12-01 15:17:01 0.065   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", true, true]
2015-12-01 15:17:01 0.001   sckoo2(1)   resolve_interpreted_items   [[{"mode": "table", "bundle_info": [{"data_hash": "0x07953d1e1c6e7e75138e8b12bdd8fcdb8e560854", "owner_name": "sckoo", "uuid": "0xb3887dd735824b96a891185e39e0b338", "args": null, "state": "ready", "dependencies": [], "bundle_type": "dataset", "owner": "sckoo(0)", "command": null, "owner_id": "0", "metadata": {"name": "hosts", "license": "", "tags": [], "created": 1443641069, "source_url": "", "data_size": 214, "description": "Upload /etc/hosts"}}, {"data_hash": "0x07953d1e1c6e7e75138e8b12bdd8fcdb8e560854", "owner_name": "sckoo", "uuid": "0xdc6f39cbfce3439b91848c86ab3fc0a1", "args": null, "state": "ready", "dependencies": [], "bundle_type": "dataset", "owner": "sckoo(0)", "command": null, "owner_id": "0", "metadata": {"name": "hosts", "license": "", "tags": [], "created": 1443641075, "source_url": "", "data_size": 214, "description": "Upload /etc/hosts"}}, {"data_hash": "0x07953d1e1c6e7e75138e8b12bdd8fcdb8e560854", "owner_name": "sckoo", "uuid": "0x08d4ea624d6944ddba585d0d27e95876", "args": null, "state": "ready", "dependencies": [], "bundle_type": "dataset", "owner": "sckoo(0)", "command": null, "owner_id": "0", "metadata": {"name": "hosts", "license": "", "tags": [], "created": 1443641081, "source_url": "", "data_size": 214, "description": "Upload /etc/hosts"}}, {"data_hash": "0x07953d1e1c6e7e75138e8b12bdd8fcdb8e560854", "owner_name": "sckoo", "uuid": "0x8bbbf17c6ccb4f73a9a1224937622caa", "args": null, "state": "ready", "dependencies": [], "bundle_type": "dataset", "owner": "sckoo(0)", "command": null, "owner_id": "0", "metadata": {"name": "hosts", "license": "", "tags": [], "created": 1443641082, "source_url": "", "data_size": 214, "description": "Upload /etc/hosts"}}, "..."], "properties": {}, "interpreted": [["uuid", "name", "summary", "state", "..."], [{"desc.": "Upload /etc/hosts", "summary": "[uploaded]", "uuid": "0xb3887d", "name": "hosts", "state": "ready"}, {"desc.": "Upload /etc/hosts", "summary": "[uploaded]", "uuid": "0xdc6f39", "name": "hosts", "state": "ready"}, {"desc.": "Upload /etc/hosts", "summary": "[uploaded]", "uuid": "0x08d4ea", "name": "hosts", "state": "ready"}, {"desc.": "Upload /etc/hosts", "summary": "[uploaded]", "uuid": "0x8bbbf1", "name": "hosts", "state": "ready"}, "..."]]}]]
2015-12-01 15:17:02 0.080   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", true, true]
2015-12-01 15:17:02 0.001   sckoo2(1)   resolve_interpreted_items   [[{"mode": "table", "bundle_info": [{"data_hash": "0x07953d1e1c6e7e75138e8b12bdd8fcdb8e560854", "owner_name": "sckoo", "uuid": "0xb3887dd735824b96a891185e39e0b338", "args": null, "state": "ready", "dependencies": [], "bundle_type": "dataset", "owner": "sckoo(0)", "command": null, "owner_id": "0", "metadata": {"name": "hosts", "license": "", "tags": [], "created": 1443641069, "source_url": "", "data_size": 214, "description": "Upload /etc/hosts"}}, {"data_hash": "0x07953d1e1c6e7e75138e8b12bdd8fcdb8e560854", "owner_name": "sckoo", "uuid": "0xdc6f39cbfce3439b91848c86ab3fc0a1", "args": null, "state": "ready", "dependencies": [], "bundle_type": "dataset", "owner": "sckoo(0)", "command": null, "owner_id": "0", "metadata": {"name": "hosts", "license": "", "tags": [], "created": 1443641075, "source_url": "", "data_size": 214, "description": "Upload /etc/hosts"}}, {"data_hash": "0x07953d1e1c6e7e75138e8b12bdd8fcdb8e560854", "owner_name": "sckoo", "uuid": "0x08d4ea624d6944ddba585d0d27e95876", "args": null, "state": "ready", "dependencies": [], "bundle_type": "dataset", "owner": "sckoo(0)", "command": null, "owner_id": "0", "metadata": {"name": "hosts", "license": "", "tags": [], "created": 1443641081, "source_url": "", "data_size": 214, "description": "Upload /etc/hosts"}}, {"data_hash": "0x07953d1e1c6e7e75138e8b12bdd8fcdb8e560854", "owner_name": "sckoo", "uuid": "0x8bbbf17c6ccb4f73a9a1224937622caa", "args": null, "state": "ready", "dependencies": [], "bundle_type": "dataset", "owner": "sckoo(0)", "command": null, "owner_id": "0", "metadata": {"name": "hosts", "license": "", "tags": [], "created": 1443641082, "source_url": "", "data_size": 214, "description": "Upload /etc/hosts"}}, "..."], "properties": {}, "interpreted": [["uuid", "name", "summary", "state", "..."], [{"desc.": "Upload /etc/hosts", "summary": "[uploaded]", "uuid": "0xb3887d", "name": "hosts", "state": "ready"}, {"desc.": "Upload /etc/hosts", "summary": "[uploaded]", "uuid": "0xdc6f39", "name": "hosts", "state": "ready"}, {"desc.": "Upload /etc/hosts", "summary": "[uploaded]", "uuid": "0x08d4ea", "name": "hosts", "state": "ready"}, {"desc.": "Upload /etc/hosts", "summary": "[uploaded]", "uuid": "0x8bbbf1", "name": "hosts", "state": "ready"}, "..."]]}]]
2015-12-01 15:17:15 0.059   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", true, true]
2015-12-01 15:17:16 0.057   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", true, true]
2015-12-01 15:17:17 0.056   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", true, true]
2015-12-01 15:17:17 0.058   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", true, true]
2015-12-01 15:17:19 0.060   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", true, true]
2015-12-01 15:17:34 0.056   sckoo2(1)   get_worksheet_info  ["0x026f9e00792943c1b16e199444091cd2", true, true]
2015-12-01 15:17:44 0.014   sckoo2(1)   search_worksheets   [[""]]
2015-12-01 15:17:45 0.013   sckoo2(1)   search_worksheets   [[""]]
```

@percyliang What exactly is happening for you?

**Comment by kashizui** [comment]
I think it might be a Python version difference (i'm on 2.7.10), the problem arises from having two arguments with the same storage variable ('command'). My version seems to let the sub-command parser override, and 2.7.6 on ocean works the other way.

**Comment by kashizui** [comment]
Pushing a fix in a second.

**Comment by kashizui** [comment]
Fixed by #274
